deck.md

```diff
@@ -346,9 +380,28 @@ export default async function users(fastify) {
 
 ---
 
+# Step 4: Trying it out
```

Comment:
was this somewhere before? Or you added it?

Reply:
I added this

I think each step should have a 'try it out' step so the class can see what should be happening after each change. I think this also works nicely for steps which should still work the same (step 8)